### ISSUE TITLE
Calling `.start()` does not appear to work with the `startManually` option

### ISSUE THREAD

**Issue by bcarriereupgrade** [SupremeTechnopriest/react-idle-timer#283]
### Affected Module

- `Version`: 5.4.2

### Describe the bug

(Seems be the same or related to issue #276)

When using the `startManually` option _in a test scenario_ the idle timer does not appear to start after calling the `.start()` method. Am testing in "real life" and will add detail.

The workaround works:
```js
      setTimeout(() => {
        idleTimer.start();
      });
```

### To Reproduce

Source code:

```js
// Edited from my code to include just the basics.
const USE_IDLE_TIMER = Boolean(Number(REACT_APP_IDLE_DETECTION_TIMEOUT || 0));

const App = ({ history, auth, loggedIn }) => {
  const idleTimer = useIdleTimer({
    startManually: true,
    crossTab: false,
    onIdle: () => {
        auth.logout(`You've been logged out after ${IDLE_TIMEOUT} minutes of inactivity`);
    },
    timeout: 1000 * 60 * IDLE_TIMEOUT
  });

  if (loggedIn) {
    if (USE_IDLE_TIMER) {
      setTimeout(() => {
        idleTimer.start();
      });
    }

    return (
      <SomeProvider>
        <Router {...{ history }} />
      </SomeProvider>
    );
  }

  return <Login {...{ history }} />;
};
```
Test:

Test is in plain Jest, no testing library, real timers. I did do the test environment setup as per the documentation.

```js
it('uses an idle timer to logout the agent automatically', async () => {
  const store = createStore({
    user: {
      loggedIn: true
    }
  });

  create(
    <Provider store={store}>
      <App auth={auth} />
    </Provider>
  );

  await sleep(1000);
  expect(auth.logout).toHaveBeenCalled();
});

```

### Expected behavior

Timer should have started and the `onIdle` callback should have been invoked given enough time.

### System Information (please complete the following information)

 - `OS`: MacOS
 - `Device`: MBP
 - `Browser Vendor`: n/a
 - `Browser Version`: n/a

**Comment by SupremeTechnopriest** [comment]
Hey @bcarriereupgrade,

Im on vacation until next Wednesday.  I will take a look at this when I get back.  If you can make a nodejs based stackblitz (so I can run the tests) that reproduces the problem that would be a big help in debugging.

**Comment by SupremeTechnopriest** [comment]
I will see what I can do about the timeout requirement.  There is a race condition somewhere.

**Comment by SupremeTechnopriest** [comment]
So I assume you are using react-router here? Is there a reason why you aren't using the router to handle the authentication redirect?  I would consider looking into that and using an authenticated wrapper to house the Idle timer instance.

Either way, this bug is fixed in IdleTimer.  It will be published in the upcoming `5.5.0` release.  If you want to test it out early you an install the release candidate version with `npm i react-idle-timer@next`.
